packages/three-vrm-springbone/src/utils/lowestCommonAncestor.ts

```diff
@@ -0,0 +1,25 @@
+import type * as THREE from 'three';
+
+/**
+ * Finds the lowest common ancestors of the given objects, if it exists.
+ * @param objects The objects to find the lowest common ancestor for.
+ */
+export function lowestCommonAncestor(objects: THREE.Object3D[]): THREE.Object3D | null {
```

Comment:
Hadn't heard of Lowest Common Ancestor but it seems it's a pretty common term in graph theory : 1: